Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM node:8-alpine
+
+# Install installation dependancies.
+RUN apk --no-cache add git
+
+# Create app directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Setup the environment.
+ENV NODE_ENV production
+
+# Bundle app source
+COPY . /usr/src/app
+
+# Install app dependencies and build static assets.
+RUN npm install && \
+  npm run compile && \
+  npm run build && \
+  npm prune --production
```

Comment:
dev dependencies were not installed for me, but this would work:
```dockerfile
# Install app dependencies and build static assets.
RUN npm install --only=production && \
  npm install --only=dev && \
  npm run compile && \
  npm run build && \
  npm prune --production
```

Reply:
Fixing this and it works great!